### ISSUE TITLE
Freezes after some time on Ubuntu 12.04 32 bit

### ISSUE THREAD

**Issue by rkfg** [alex2108/syncthing-tray#10]
I don't see a pattern here, it just works for some time and then stops updating, tray menu also doesn't work. Log doesn't contain anything useful just state changes. Works fine on my Debian jessie amd64 though.

**Comment by alex2108** [comment]
sounds strange, and I don't know if I can do anything then. I only know the bug that the text on the menu is not updated on Ubuntu which is probably a issue of trayhost. Do you still get state changes after the icon and menu are frozen?

**Comment by rkfg** [comment]
Nope, it stops completely. The process just sits there and doesn't react to anything including Syncthing state changes.
